specs-jetpack-calypso/wp-pressable-nux-spec.js

```diff
@@ -0,0 +1,92 @@
+import test from 'selenium-webdriver/testing';
+import config from 'config';
+import assert from 'assert';
+
+import PressableLogonPage from '../lib/pages/pressable/pressable-logon-page';
+import PressableSitesPage from '../lib/pages/pressable/pressable-sites-page';
+import PressableApprovePage from '../lib/pages/pressable-approve-page';
+import PressableSiteSettingsPage from '../lib/pages/pressable/pressable-site-settings-page';
+import JetpackAuthorizePage from '../lib/pages/jetpack-authorize-page';
+import WPAdminDashboardPage from '../lib/pages/wp-admin/wp-admin-dashboard-page';
+
+import * as driverManager from '../lib/driver-manager';
+import * as dataHelper from '../lib/data-helper';
+import LoginPage from '../lib/pages/login-page';
+import PressableNUXFlow from '../lib/flows/pressable-nux-flow';
+
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
+const screenSize = driverManager.currentScreenSize();
+const host = dataHelper.getJetpackHost();
+
+var driver;
+
+test.before( function() {
+	this.timeout( startBrowserTimeoutMS );
+	driver = driverManager.startBrowser();
+} );
+
+test.describe( `[${host}] Pressable NUX: (${screenSize}) @jetpack`, function() {
+	this.timeout( mochaTimeOut );
+
+	test.describe( 'Flow', function() {
+		this.bailSuite( true );
+
+		test.before( function() {
+			return driverManager.clearCookiesAndDeleteLocalStorage( driver );
+		} );
+
+		test.it( 'Can log into Pressable', function() {
+			this.pressableLoginPage = new PressableLogonPage( driver, true );
+			return this.pressableLoginPage.loginWithWP();
+		} );
+
+		test.it( 'Can log into WordPress.com', function() {
+			const user = dataHelper.getAccountConfig( 'jetpackUserPRESSABLE' );
+			this.loginPage = new LoginPage( driver );
+			return this.loginPage.login( user[0], user[1] );
+		} );
+
+		test.it( 'Can approve login with WordPress', function() {
+			this.pressableApprovePage = new PressableApprovePage( driver );
+			return this.pressableApprovePage.approve();
+		} );
+
+		test.it( 'Can create new site', function() {
+			this.siteName = dataHelper.getNewBlogName();
+			this.pressableSitesPage = new PressableSitesPage( driver );
+			return this.pressableSitesPage.addNewSite( this.siteName );
```

Comment:
My test run failed at this point because the Pressable test account already has the maximum allowed sites (5 sites), so at least one site will need to be deleted to create a new one. A couple ideas:

- We could check for that before adding a new site, and delete one if needed.
- We could clean up at the end of the flow and delete the new site we created.

Reply:
I see now that you do clean up at the end of the flow, so I guess we also need a check just in case that doesn't work as expected and the account gets filled up.